content/influxdb/v2.0/reference/cli/influx/auth/list.md

```diff
@@ -21,21 +21,22 @@ influx auth list [flags]
-| `-i` | `--id`            | Authentication token ID                                               | string      |                       |
-|      | `--json`          | Output data as JSON (default `false`)                                 |             | `INFLUX_OUTPUT_JSON`  |
-| `-o` | `--org`           | Organization name (mutually exclusive with `--org-id`)                | string      |                       |
-|      | `--org-id`        | Organization ID (mutually exclusive with `--org`)                     | string      |                       |
-|      | `--skip-verify`   | Skip TLS certificate verification                                     |             |                       |
-| `-t` | `--token`         | Authentication token                                                  | string      | `INFLUX_TOKEN`        |
-| `-u` | `--user`          | Username                                                              | string      |                       |
-|      | `--user-id`       | User ID                                                               | string      |                       |
+| Flag |                   | Description                                                           | Input type | {{< cli/mapped >}}    |
+| :--- | :---------------- | :-------------------------------------------------------------------- | :--------: | :-------------------- |
+| `-c` | `--active-config` | CLI configuration to use for command                                  |   string   |                       |
+|      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`) |   string   | `INFLUX_CONFIGS_PATH` |
+| `-h` | `--help`          | Help for the `list` command                                           |            |                       |
+|      | `--hide-headers`  | Hide table headers (default `false`)                                  |            | `INFLUX_HIDE_HEADERS` |
+|      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)            |   string   | `INFLUX_HOST`         |
+|      | `--http-debug`    | Inspect communication with InfluxDB servers.                          |   string   |                       |
+| `-i` | `--id`            | Authentication token ID                                               |   string   |                       |
+|      | `--json`          | Output data as JSON (default `false`)                                 |            | `INFLUX_OUTPUT_JSON`  |
+| `-o` | `--org`           | Organization name (mutually exclusive with `--org-id`)                |   string   |                       |
+|      | `--org-id`        | Organization ID (mutually exclusive with `--org`)                     |   string   |                       |
+|      | `--skip-verify`   | Skip TLS certificate verification                                     |            |                       |
```

Comment:
i think skip-verify also has an environment variable now as of https://github.com/influxdata/influx-cli/pull/208 unless that's addressed in another PR.

Reply:
It's in #2944